lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -1056,30 +1081,19 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
   }
 
   if (auto triu = dyn_cast<AtenTriuOp>(op)) {
-    // Check if the rank of the input tensor is valid.
-    AtenTriuOp::Adaptor adaptor(operands);
-    auto inputType = adaptor.getSelf().getType().cast<RankedTensorType>();
-    uint64_t inputRank = inputType.getRank();
-    if (inputRank < 2) {
-      triu.emitError("too few dimensions to compute triangular part of matrix");
```

Comment:
This explicit condition is not needed because this op by definition fails on an input tensor with rank < 2.